### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by MAleemH**
- Set up a new project with webpack
- Created an array for dynamic data output
- Wrote a function in javaScript to iterate over the tasks array
- Gave some CSS for better outlook

### CODE REVIEW

**Review by Ismailco — COMMENTED** [review]
Hi @MAleemH, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no professional documentation, kindly make sure to use the README provided by Microverse

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._https://github.com/MAleemH/to-do-list/pull/1

File: package.json [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+    "wrappy": "^1.0.2",
+    "write-file-atomic": "^3.0.3",
+    "yallist": "^3.1.1",
+    "yaml": "^1.10.2",
+    "yargs-parser": "^20.2.9",
+    "zwitch": "^1.0.5"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```
Comment (Davidosky007) [inline package.json]: * After running the build command I get a blank screen when I open the /dist/index.html file on the browser, Kindly fix this and also add more webpack script with setup instructions on your readme file on how to run your app
image
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @MAleemH 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* No linter errors 👍 
* Proper use of Github flow 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (MAleemH) [reply package.json]: Hi @Davidosky007!
I hope you are doing great. I have tried myself but unable to fix this. I'm unable to know that how to push my src html file to dist file. If you have some free time than please help me to figure out the solution. Thanks.

**Review by levy002 — COMMENTED** [review]
## Hello @MAleemH :wave 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle: 

You have submitted a project, that doesn't run linters.

- [ ] Kindly make sure to set up linters correctly so it runs to check if there are no errors on your pull request. I recommend you take a step back and read this[ lesson](https://github.com/microverseinc/linters-config/blob/master/html-css-js) to understand it clearly and follow the guideline to set it up correctly.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by topeogunleye — APPROVED** [review]
### Hi @MAleemH, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good summary description :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**